docs/customization/form.rst

```diff
@@ -19,6 +19,11 @@ You can:
 How to customize a Form?
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. tip::
+
+    You can browse the full implementation of this example on GitHub
+    on the `<https://github.com/Sylius/Customizations/pull/8>`_
```

Comment:
How does this look like when you render it?

Reply:
I added picture in my PR. It is not enough?